api/features/workflows/core/models.py

```diff
@@ -35,11 +35,11 @@
     create_feature_state_updated_by_change_request_audit_log,
     create_feature_state_went_live_audit_log,
 )
+from environments.tasks import rebuild_environment_document
 from features.models import FeatureState
+from features.versioning.models import EnvironmentFeatureVersion
 from features.versioning.tasks import trigger_update_version_webhooks
-
-from ...versioning.models import EnvironmentFeatureVersion
-from .exceptions import (
+from features.workflows.core.exceptions import (
```

Comment:
In general I think absolute imports are always fine, but one level of relative imports like `from .exceptions import ...` are acceptable too. So I agree with your redefinition of `from ...versioning.models import EnvironmentFeatureVersion` since it's too many levels deep but I don't really see the importance of redefining the exceptions module. 

That said, I'm not really hung up on it, I'm more just interested in knowing why you're favouring the absolute import for exceptions here.

Reply:
Yeah, that's a fair point. I think I just like standardising to a single approach.